services/motion/builtin/builtin_test.go

```diff
@@ -552,6 +552,7 @@ func TestMoveOnMapPlans(t *testing.T) {
 	})
 
 	t.Run("check that position-only mode executes", func(t *testing.T) {
+		t.Skip()
```

Comment:
mind adding a comment to the ticket for unskipping this test?

Reply:
done

Reply:
already approved. Merge at will